src/test/java/seedu/contax/logic/commands/CommandTestUtil.java

```diff
@@ -94,6 +99,34 @@
     public static final String PREAMBLE_WHITESPACE = "\t  \r  \n";
     public static final String PREAMBLE_NON_EMPTY = "NonEmptyPreamble";
 
+    public static final String COMMAND_VALID_CSV_FILEPATH = " " + PREFIX_FILE + VALID_CSV_FILEPATH;
+    public static final String COMMAND_INVALID_BAD_EXTENSION_FILEPATH = " " + PREFIX_FILE
+            + INVALID_BAD_EXTENSION_CSV_FILEPATH;
+    public static final String COMMAND_INVALID_NO_EXTENSION_FILEPATH = " " + PREFIX_FILE
+            + INVALID_NO_EXTENSION_CSV_FILEPATH;
+    public static final String COMMAND_CSV_NAMEPOSITION = " " + PREFIX_NAME + 1;
+    public static final String COMMAND_CSV_NAMEPOSITION_CLASH = " " + PREFIX_NAME + 2;
+    public static final String COMMAND_CSV_PHONEPOSITION = " " + PREFIX_PHONE + 2;
```

Comment:
what would here and following +1 +2 mean, is that like magic num?

Reply:
it's supposed to be simulated positions, so
`COMMAND_CSV_NAMEPOSITION = " " + PREFIX_NAME + 1` will be
` n/ 1` to specify name field in column 1

the clash one is set to 2 such that it clashes with another position, causing exception

I've changed the numbers to be strings instead too to make this clearer!